test/filters/test_blur.py

```diff
@@ -148,3 +149,10 @@ def test_module(self, device, dtype):
         actual = op_module(kernel_size)(img)
         expected = op(img, kernel_size)
         assert_close(actual, expected)
+
+    @pytest.mark.onnx
+    def test_onnx_export(self, device, dtype):
+        kernel_size = (3, 3)
+        op_module = kornia.filters.BoxBlur(kernel_size)
+        img = torch.rand(2, 3, 4, 5, device=device, dtype=dtype)
+        torch.onnx.export(op_module, img, "temp.onnx", export_params=True)
```

Comment:
```suggestion
        torch.onnx.export(op_module, img, "temp.onnx", export_params=True, op_set=11)
```
I’d also look into the dynamic axes parameters 
https://pytorch.org/docs/stable/onnx.html#torch.onnx.export